tests/ui/fn/fn_def_coercion.rs

```diff
@@ -0,0 +1,57 @@
+//! Test that coercing between function items of the same function,
+//! but with different args works.
```

Comment:
the usage of the word args here is very confusing. could you disambiguate with a "generic" (or just use the unambiguous word "substs" 😉).

Reply:
you fixed the other one but not this one :3

Reply:
🤦